pkg/cmd/server/api/v1/types.go

```diff
@@ -451,6 +451,7 @@ type KubernetesMasterConfig struct {
 	ServicesSubnet      string   `json:"servicesSubnet"`
 	StaticNodeNames     []string `json:"staticNodeNames"`
 	SchedulerConfigFile string   `json:"schedulerConfigFile"`
+	PodEvictionTimeout  string   `json:"podEvictionTimeout"`
```

Comment:
need to update types_test.go to include the new field, and open a doc and config change issue with details